### DISCUSSION TITLE
Welcome to ocilib Discussions!

### DISCUSSION THREAD

**Discussion by vrogier** [vrogier/ocilib#254] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by FladoToo** [comment]
Hi, I'm Vladimir Andreev, also called Flado. I have lots of experience with development of applications using an Oracle database, performance tuning and such.
Thanks for the excellent work you're doing with OCILIB!

**Comment by vrogier** [comment]
Hello Vlad, 

It is been such a long time ;)

Thansk 👍 

Vincent
